website/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "chisel-lang",
+  "version": "0.1.0",
```

Comment:
What's the versioning story here or is this a dummy value?

Reply:
It defaulted to `0.0.0`, I just upped it to `0.1.0`, I'll see if I can forgo having a version